src/MongoDB.Driver/FindOneAndReplaceOptions.cs

```diff
@@ -54,12 +65,15 @@ public Collation Collation
         }
 
         /// <summary>
-        /// Gets or sets a value indicating whether to bypass document validation.
+        /// Gets or sets the hint.
         /// </summary>
-        public bool? BypassDocumentValidation
+        /// <value>
```

Comment:
this class doesn't have <value> xml comment in other properties

Reply:
Done.